manual/manual/refman/exten.etex

```diff
@@ -531,12 +527,10 @@ Note that "'b" is not explicitly polymorphic here, and it will
 actually be unified with "int".
 
 The other application is to ensure that some definition is sufficiently
-polymorphic.
-\caml
-\?let id : \tq\-a. \tq\-a -> \tq\-a = \<fun x -> x+1\> ;;
-\:Error: This definition has type int -> int which is less general than
-\:         \tq\-a. \tq\-a -> \tq\-a
-\endcaml
+polymorphic:
+\begin{caml_example}[error]
+let id: 'a. 'a -> 'a = fun x -> x
```

Comment:
Nice catch! Thanks.

As a side-note, the crucial error here was the missing `;;` at the end of the `caml_example`
that circumvented the status detection for the `caml_example`. This is fixed in the new version
of the PR.